content/en/blog/2024/policing-a-sandbox.md

```diff
@@ -0,0 +1,296 @@
+---
+date: 2024-08-15
+title: Policing a Sandbox
+linkTitle: Policing a Sandbox
+description: >
+  How can we provide integrity guarantees for dynamic workloads in the Confidential Containers stack?
+author: Magnus Kulke ([@mkulke](https://github.com/mkulke))
+---
+
+{{< figure src="/img/sandbox-0.png" alt="The image is a detailed black-and-white technical drawing of a sandbox area. It features a rectangular sandbox filled with sand, with a shovel and bucket inside. To the right of the sandbox, there is a lifeguard chair. Above the sandbox, there is an overhead view of a life preserver ring. In the background, there are architectural plans and measurements for constructing the sandbox area, including various dimensions and structural details." >}}
+
+In a [previous article]({{< ref "building-trust-into-os-images-for-coco.md" >}}) we discussed how we can establish confidence in the integrity of an OS image for a confidential Guest, that is supposed to host a collocated set (Pod) of confidential containers. The topic of this article will cover the integrity of the more dynamic part of a confidential container's lifecycle. We'll refer to this phase as "runtime", even though from the perspective of Container workload it might well be before the actual execution of an entrypoint or command.
+
+{{< figure src="/img/sandbox-1.png" width="65%" alt="The image is a diagram illustrating the components of a sandbox environment, divided into dynamic and static components. It uses color coding to differentiate between various elements: two blue squares labeled ‘Pod Sandbox’ and ‘Container’ represent the dynamic sandbox, while three green squares labeled ‘Base OS,’ ‘Kata Agent,’ and ‘Guest Components’ represent the static components. A legend at the bottom explains the color coding." >}}
+
+A Confidential Containers (CoCo) OS image contains static components like the Kernel and a root filesystem with a container runtime (e.g kata-agent) and auxiliary binaries to support remote attestation. We’ve seen that those can be covered in a comprehensive measurement that will remain stable across different instantiations of the same image, hosting varying Confidential Container workloads.
```

Comment:
"hosting varying Confidential Container workloads."

This part is not so easy to parse without having to re-read it.

What about "hosting a variety of confidential containers workloads"?